packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -523,6 +524,83 @@ export const trackEventsMapper = {
     wishlistId: data.properties?.wishlistId,
     lineItems: getProductLineItems(data),
   }),
+  [eventTypes.PRODUCT_UPDATED]: data => {
+    const eventList = [];
+    const properties = data.properties;
+
+    if (properties.colour && properties.oldColour !== properties.colour) {
+      // color changed event
+      const additionalParameters = {};
+      if (properties?.oldColourId && properties?.colourId) {
+        additionalParameters[
+          'colourList'
+        ] = `${properties?.oldColourId}, ${properties?.colourId}`;
+      } else {
+        logger.warn(
+          `[Omnitracking] - Event ${data.event} property oldColourId and colourId can be used on track.
```

Comment:
I would update this warning to:

```suggestion
          `[Omnitracking] - Event ${data.event} properties "oldColourId" and "colourId" should be sent on the payload when triggering a "change color" event. If you want to track this event, make sure to pass these two properties.`,
```
What do you think?

Reply:
Updated!